apps/pipelines/models.py

```diff
@@ -379,19 +380,23 @@ def _save_message_to_history(
         return chat_message
 
     @transaction.atomic()
-    def create_new_version(self):
+    def create_new_version(self, is_copy: bool = False):
         version_number = self.version_number
-        self.version_number = version_number + 1
+        self.version_number = 1 if is_copy else self.version_number + 1
```

Comment:
This will update the version of the item being copied which is incorrect:
```suggestion
        if not is_copy:
            self.version_number = self.version_number + 1
```